### PR TITLE
fix: various UI fixes from review session

### PR DESCRIPTION
**Pull request by ZachJW34**
- Closes [UNIFY-679](https://cypress-io.atlassian.net/browse/UNIFY-679)

### User facing changelog
Remove 'v' from browser versions, add cursor to card, refactor reporter stats and controls styles

### Additional details
Some various UI changes while reviewing the UI with @ryanjwilke @brian-mann. Didn't get through all of the UI we wanted to go over, so more to come in another PR.

### How has the user experience changed?
Rearranged the reporter stats and aligned the content

https://user-images.githubusercontent.com/25158820/146617575-5e4fea43-9e24-413d-9b22-996297af1563.mov

### CODE REVIEW

File: packages/frontend-shared/windi.config.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const defaultConfig: WindiCssOptions = {
     },
     cursor: {
       'ew-resize': 'ew-resize',
+      'pointer': 'pointer',
```
Comment (elevatebart) [inline packages/frontend-shared/windi.config.ts:26]: Can you comment on why this change is necessary?
I would feel that this class exists by default.

**Review by elevatebart — COMMENTED** [review]
Have you checked/reviewed the "pause" screen too?

It seems misaligned:
Screen Shot 2021-12-20 at 11 04 50 AM

File: packages/frontend-shared/windi.config.ts (hunk 1 above)
Comment (ZachJW34) [reply packages/frontend-shared/windi.config.ts:26]: Found the issue, `cursor: { ... }` should be under the `extend` property. Since it wasn't, it was removing all of the default cursor properties. Moving it under `extends` fixes this.

**Comment by ZachJW34** [comment]
@elevatebart good catch, I didn't check that

**Comment by ZachJW34** [comment]
@elevatebart updated and fixed the controls layout

https://user-images.githubusercontent.com/25158820/146811018-c0d355c9-6dda-47bf-af56-f86b1a43678c.mov

File: packages/launchpad/src/setup/OpenBrowserList.vue [hunk 2]
```diff
@@ -35,20 +35,20 @@
...
               :src="allBrowsersIcons[browser.displayName]"
               :alt="browser.displayName"
-              class="inline w-40px h-40px"
+              class="h-40px w-40px inline"
             >
           </div>
-          <div class="pt-2 text-lg text-indigo-600">{{ browser.displayName }}</div>
-          <div class="text-xs text-gray-400">v{{ browser.majorVersion }}.x</div>
+          <div class="text-lg pt-2 text-indigo-600">{{ browser.displayName }}</div>
+          <div class="text-xs text-gray-400">v{{ browser.majorVersion }}</div>
```
Comment (marktnoonan) [inline packages/launchpad/src/setup/OpenBrowserList.vue:42]: We removed the `v` in the other list, did we want to also remove it here?

Comment (ZachJW34) [reply packages/launchpad/src/setup/OpenBrowserList.vue:42]: Confirmed with Ryan, he wants the `v` to stay in this case.

**Review by elevatebart — APPROVED** [review]
LGTM
